src/utils/config.ts

```diff
@@ -44,7 +44,7 @@ export class Config extends events.EventEmitter {
       Config.getInstance().loginUrl = organization.get<string>('loginUrl');
       Config.getInstance().username = organization.get<string>('username');
       Config.getInstance().password = organization.get<string>('password');
-      Config.getInstance().securityToken = organization.get<string>('securityToken');
+      Config.getInstance().securityToken = organization.get<string>('securityToken') || "";
```

Comment:
You should instead change the default value for the `securityToken` option inside the package.json file
```
"vsforce.organization.securityToken": {
    "type": "string",
    "default": null,
    "description": "Your Salesforce organization user security token"
}
```